cvmfs/cache_posix.h

```diff
@@ -73,7 +73,7 @@ class PosixCacheManager : public CacheManager {
     const std::string &cache_path,
     const bool alien_cache,
     const RenameWorkarounds rename_workaround = kRenameNormal,
-    const bool do_refcount = false);
+    const bool do_refcount = true);
```

Comment:
the following lines need the same change:
 https://github.com/cvmfs/cvmfs/blob/64595763e066911ed28332dfb92bfaf7395e72de/cvmfs/cache_posix.h#L142-L143

Reply:
Good catch, thanks!